docs/guidance/model/milestones.md

```diff
@@ -0,0 +1,52 @@
+# Milestones
+
+Milestones can be included within the planning, tender, contract and contract implementation sections. 
+
+* The planning milestone block may describe anticipated milestones during the planning stage, such as the preparation 
+of key studies. Usually using the 'preProcurement' milestone type code.
+
+* The tender milestone block should be used to describe anticipated milestones during the proposed delivery 
+of a contract, as well as key dates from the process of tendering and awarding the contract.
+
+* The contract milestone block should be used for events related to the signing of the contract documents.
+
+* The contract implementation milestone block should be used for any event related to delivery of the contract.
+
+The nature of the milestone is indicated by the [milestone type codelist](../../schema/codelists#milestone-type).
+
+The lifecycle of the milestone should be modeled with the fields: `dueDate`, `dateMet`, `dateModified` and [status](../../schema/codelists/#milestone-status).
+
+## Worked example
+
+In the example below:
```

Comment:
maybe "example" -> "examples" to make it clearer that people can switch between the 3 cases? (unless it's intended to be one example!)

And add some explanation on whether 2 and 3 are different possible examples following on from the first (see comments on json files, above).